apps/meteor/app/otr/client/rocketchat.otr.room.js

```diff
@@ -52,8 +61,6 @@ OTR.Room = class {
 	}
 
 	acknowledge() {
-		APIClient.v1.post('statistics.telemetry', { params: [{ eventName: 'otrStats', timestamp: Date.now(), rid: this.roomId }] });
```

Comment:
it's a product decision to remove this one? 🤔

Reply:
Removed by mistake, maybe while solving monorepo conflicts.